src/main/java/com/coreos/jetcd/KV.java

```diff
@@ -20,36 +20,72 @@
 @Beta
 public interface KV {
 
-  // ***************
-  // Op.PUT
-  // ***************
-
+  /**
+   * put a key-value pair into etcd.
+   *
+   * @param key key in ByteSequence
+   * @param value value in ByteSequence
+   * @return PutResponse
+   */
   CompletableFuture<PutResponse> put(ByteSequence key, ByteSequence value);
 
+  /**
+   * put a key-value pair into etcd.
+   *
+   * @param key key in ByteSequence
+   * @param value value in ByteSequence
+   * @param option PutOption
+   * @return PutResponse
+   */
   CompletableFuture<PutResponse> put(ByteSequence key, ByteSequence value, PutOption option);
 
-  // ***************
-  // Op.GET
-  // ***************
-
+  /**
+   * retrieve keys. By default, get will return the value for "key".
+   *
+   * @param key key in ByteSequence
+   * @return RangeResponse
+   */
   CompletableFuture<RangeResponse> get(ByteSequence key);
 
+  /**
+   * retrieve keys. By default, get will return the value for "key".
+   *
+   * @param key key in ByteSequence
+   * @param option GetOption
+   * @return RangeResponse
+   */
   CompletableFuture<RangeResponse> get(ByteSequence key, GetOption option);
 
-  // ***************
-  // Op.DELETE
-  // ***************
-
+  /**
+   * delete a key.
+   *
+   * @param key key in ByteSequence
+   * @return DeleteRangeResponse
+   */
   CompletableFuture<DeleteRangeResponse> delete(ByteSequence key);
 
+  /**
+   * delete a key.
+   *
+   * @param key key in ByteSequence
+   * @param option DeleteOption
+   * @return DeleteRangeResponse
+   */
   CompletableFuture<DeleteRangeResponse> delete(ByteSequence key, DeleteOption option);
 
-  // ***************
-  // Op.COMPACT
-  // ***************
-
+  /**
+   * compact etcd KV history before the given rev.
+   *
+   * @return CompactionResponse
+   */
   CompletableFuture<CompactionResponse> compact();
```

Comment:
good catch. the compact API should take at least one arg which is the rev to be compacted at. I'll fix this.